pkg/command/login.go

```diff
@@ -374,7 +374,7 @@ func globalLoginUsingServer(s *configtypes.Server) (err error) {
 		issuer = csp.StgIssuer
 	}
 	if apiTokenExists {
-		log.Debug("API token env var is set")
+		log.Info("API token env var is set")
```

Comment:
Same case as for `context`, in case we decide to change anything in `context.go`